### ISSUE TITLE
Packagekit fails with missing gpg key

### ISSUE THREAD

**Issue by camckay** [AlmaLinux/raspberry-pi#19]
Actual error output...

`Command failed: Failed to download gpg key for repo 'raspberrypi': Curl error (37): Couldn't read a file:// file for file:///etc/pki/rpm-gpg/RPM-GPG-KEY-AlmaLinux [Couldn't open file /etc/pki/rpm-gpg/RPM-GPG-KEY-AlmaLinux]`

/etc/pki/rpm-gpg/RPM-GPG-KEY-AlmaLinux doesn't actually exist. Should be looking for RPM-GPG-KEY-AlmaLinux-9. Creating symlink works around the issue.

**Comment by metalefty** [comment]
@srbala I've created PR. https://git.almalinux.org/rpms/almalinux-release-raspberrypi/pulls/1

**Comment by metalefty** [comment]
The fixed package has been released. I'll test `dnf update` when it spread spread to mirrors.

**Comment by andrewlukoshko** [comment]
It's also possible to fix this by importing GPG key manually:
```
rpm --import /etc/pki/rpm-gpg/RPM-GPG-KEY-AlmaLinux-9
```
I'll close this issue when we have updated image.

**Comment by metalefty** [comment]
This will also happen when executing not-installed commands:
```
[meta@rpi400 ~]$ tig
bash: tig: command not found...
Failed to search for file: Failed to download gpg key for repo 'raspberrypi': Curl error (37): Couldn't read a file:// file for file:///etc/pki/rpm-gpg/RPM-GPG-KEY-AlmaLinux [Couldn't open file /etc/pki/rpm-gpg/RPM-GPG-KEY-AlmaLinux]
```

Now we have a fixed package. It is now available via `dnf update`. After updating package, the issue no longer happens.

```
[meta@rpi400 ~]$ tig
bash: tig: command not found...
Install package 'tig' to provide command 'tig'? [N/y]
```

**Comment by metalefty** [comment]
Now the issue is solved at 20230615 image.
https://repo.almalinux.org/development/almalinux/9/raspberrypi/
